app-office/planner/planner-0.14.6_p20130520-r3.ebuild

```diff
@@ -1,32 +1,35 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
```

Comment:
Updating EAPI means you need to adapt to the EAPI changes as well. In this case I see at least that build dependencies aren't BDEPEND but kept as DEPEND.